src/patternfly/components/Switch/switch.scss

```diff
@@ -28,6 +27,7 @@
   --pf-c-switch__input--focus__toggle--OutlineColor: var(--pf-global--primary-color--100);
 
   // Switch toggle
+  --pf-c-switch__toggle--Height: var(--pf-c-switch--Height);
```

Comment:
Can this be updated to match the value of the `--pf-c-switch--Height` var? Since `--pf-c-switch--Height` can be used to set the value of `--pf-c-switch__toggle--Height` now, if we choose to give `--pf-c-switch__toggle--Height` a custom value at some point, and `--pf-c-switch--Height` no longer sets the toggle height, that would technically be a breaking change.

Reply:
I updated it, a lot of the toggle vars rely on `pf-c-switch--Height` so I'm wondering if I should update all of them to represent the value? But at the same time some of them should be based on the height of the switch. I could see the toggle being a var that is potentially changed though so im in favor of updating that.

but something like: `--pf-c-switch__toggle--BorderRadius: var(--pf-c-switch--Height);` doesn't really need to rely on the switch's height

Reply:
Yeah good call, not sure why that is set. That could just be `var(--pf-global--BorderRadius--lg)`.